test/go-tests/test_provisioningURL.go

```diff
@@ -0,0 +1,161 @@
+const baseProjectPath = "/controlPlane/v1/project"
+
+var provisioningConfigMapTemplate = `apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mockserver-config
+data:
+  initializerJson.json: |-
+    [
+      {
+        "httpRequest": {
+          "path": "/repository",
+          "method": "POST"
+        },
+        "httpResponse": {
+          "body": {
+            "gitRemoteURL": "http://gitea-http:3000/%s/url-provisioning",
+            "gitToken":     "%s",
+            "gitUser":      "%s"
+          },
+          "statusCode": 201
+        }
+      },
+      {
+        "httpRequest": {
+          "path": "/repository",
+          "method": "DELETE"
+        },
+        "httpResponse": {
+          "body": "Repository deleted",
+          "statusCode": 200
+        }
+      },
+    ]
+  mockserver.properties: |-
+    ###############################
+    # MockServer & Proxy Settings #
+    ###############################
+    # Socket & Port Settings
+    # socket timeout in milliseconds (default 120000)
+    mockserver.maxSocketTimeout=120000
+    # Certificate Generation
+    # dynamically generated CA key pair (if they don't already exist in
+    specified directory)
+    mockserver.dynamicallyCreateCertificateAuthorityCertificate=true
+    # save dynamically generated CA key pair in working directory
+    mockserver.directoryToSaveDynamicSSLCertificate=.
+    # certificate domain name (default "localhost")
+    mockserver.sslCertificateDomainName=localhost
+    # comma separated list of ip addresses for Subject Alternative Name domain
+    names (default empty list)
+    mockserver.sslSubjectAlternativeNameDomains=www.example.com,www.another.com
+    # comma separated list of ip addresses for Subject Alternative Name ips
+    (default empty list)
+    mockserver.sslSubjectAlternativeNameIps=127.0.0.1
+    # CORS
+    # enable CORS for MockServer REST API
+    mockserver.enableCORSForAPI=true
+    # enable CORS for all responses
+    mockserver.enableCORSForAllResponses=true
+    # Json Initialization
+    mockserver.initializationJsonPath=/config/initializerJson.json
+`
+
+func Test_ProvisioningURL(t *testing.T) {
```

Comment:
Can we please start adding some annotations to describe the purpose of the integration tests?
```
//@Description: Description of what is supposed to test 
//@Outcome Expected outcome of the test
//@Issue Issue number 
```

Reply:
Sure, added, thanks :)